src/runtime/test/manifest-test.ts

```diff
@@ -1070,7 +1075,8 @@ ${particleStr1}
     const manifest = await Manifest.load('the.manifest', loader);
     const store = manifest.findStoreByName('Store0');
     assert(store);
-    assert.deepEqual(await store.toList(), [
+
+    assert.deepEqual(await store['toList'](), [
```

Comment:
Add a TODO(shans) to ensure these references get fixed as part of the storage refactor. In general I strongly want to avoid using this syntax as it's hard to read and not easily findable using searches.

Reply:
Right, we need collection- and variable-specific interfaces that provide the differing methods. It's definitely something I've been thinking about in the context of the refactor.